e2e/scenarios/create-subgraphs.test.ts

```diff
@@ -0,0 +1,72 @@
+import { expect } from 'chai'
+import hre from 'hardhat'
+import { recreatePreviousSubgraphId } from './lib/subgraph'
+import { BigNumber } from 'ethers'
+import { CuratorFixture, getCuratorFixtures } from './fixtures/curators'
+import { SubgraphFixture, getSubgraphFixtures, getSubgraphOwner } from './fixtures/subgraphs'
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers'
+
+let curatorFixtures: CuratorFixture[]
+let subgraphFixtures: SubgraphFixture[]
+let subgraphOwnerFixture: SignerWithAddress
+
+describe('Publish subgraphs', () => {
+  const { contracts, getTestAccounts } = hre.graph()
+  const { GNS, GraphToken, Curation } = contracts
+
+  before(async () => {
+    const testAccounts = await getTestAccounts()
+    curatorFixtures = getCuratorFixtures(testAccounts)
+    subgraphFixtures = getSubgraphFixtures()
+    subgraphOwnerFixture = getSubgraphOwner(testAccounts).signer
+  })
+
+  describe('GRT balances', () => {
+    it(`curator balances should match airdropped amount minus signalled`, async function () {
+      for (const curator of curatorFixtures) {
+        const address = curator.signer.address
+        const balance = await GraphToken.balanceOf(address)
+        expect(balance).gte(curator.grtBalance.sub(curator.signalled))
```

Comment:
Same as my other comment, should this be `.eq`?

Reply:
Same as above.